velox/connectors/hive/iceberg/IcebergSplitReaderBenchmark.cpp

```diff
@@ -0,0 +1,387 @@
+/*
```

Comment:
The benchmarks and tests are usually put in the tests folder. Will you please move them to velox/connectors/hive/iceberg/tests?